index.html

```diff
@@ -203,6 +203,23 @@ <h1 class="about-title">About Myself</h1>
                 </ul>
             </div>
         </section>
+        <section class="contact-secion">
+            <div class="contact">
+                <div class="contact-info">
+                    <h2>Contact me</h2>
+                    <p>if you have an application you are intersted in developing, a feature you that you need built or a project that needs coding. I'd love to help with it!</p>
+                </div>
+                <form class="contact-form" action="https://formspree.io/f/{form_id}" method="post">
+                    <input class="form-style" type="text" name="name" id="name" placeholder="Enter your name" maxlength="30" required>
+                    <input class="form-style" type="email" name="email" id="email" placeholder="Enter your email address" required>
+                    <textarea class="form-style" name="message" id="message" cols="30" rows="10" maxlength="500" placeholder="Write your message here" required></textarea>
+                    <div>
+                        <input class="regular-btn get-in-touch" type="submit" name="submit" id="submit" value="Get In Touch">
```

Comment:
- [ ] Great job so far @Esmatullah12. Your project looks great. However, I noticed that your `Submit` button is missing the button interaction styles defined for your template. So kindly include this as well to improve user interactivity with your project :+1: 
- Button styles
![image](https://user-images.githubusercontent.com/102699958/228328065-a8eeaafc-0c54-4ee0-9bd0-6e15824ec08b.png)